commands/outliers.js

```diff
@@ -19,14 +19,18 @@ function * run (context, heroku) {
     ? 'CASE WHEN length(query) <= 40 THEN query ELSE substr(query, 0, 39) || \'…\' END'
     : 'query'
 
+  let limitMatch = /^(\d+)$/.exec(context.flags.limit)
+  let limit = limitMatch ? parseInt(limitMatch[1]) : 10
```

Comment:
Come to think of it, can you change this to complain if the limit does not parse as an integer. Right now if you pass in `--limit lol` it will generate a query with `LIMIT NaN` I think, which is bogus.

Reply:
Oh no, never mind--the whole `limitMatch` is null so it'll just be `LIMIT 10`. But that's also surprising and probably not what we want.

Reply:
Do we want a nice error displayed? Any suggestions on what would be most useful?

Reply:
```
$ heroku pg:outliers -n wow -a APPNAME DATABASE
 ▸    Cannot parse num param value "wow" to a number
```